plugin.yaml

```diff
@@ -0,0 +1,69 @@
+##################################################################################
+# Cloudify host pool built in types and plugins definitions.
+##################################################################################
+
+plugins:
+    hostpool:
+        executor: central_deployment_agent
+        source: https://github.com/cloudify-cosmo/cloudify-host-pool-plugin/archive/master.zip
+
+node_types:
+  host_pool_node:
+    derived_from: cloudify.nodes.Compute
+    interfaces:
+      cloudify.interfaces.worker_installer:
+        install:
+          implementation: agent_installer.worker_installer.tasks.install
+          inputs:
+            cloudify_agent: 
+              default:
+                user: {get_attribute: [SELF, cloudify_agent, user]}
```

Comment:
this should transform to:

    user: {get_attribute: [SELF, username]}

which will also make it consistent with the softlayer plugin:
https://github.com/cloudify-cosmo/cloudify-softlayer-plugin/blob/master/plugin.yaml#L174